msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase833511.java

```diff
@@ -0,0 +1,70 @@
+package com.microsoft.identity.client.msal.automationapp.testpass.broker;
```

Comment:
Add License

Reply:
How do I add a license?

Reply:
The license is a comment block at the top of every class in our library, it's a statement of Microsoft's Copyright and ownership of each class, we can't add a class to our library without including it. You can find it in any other existing class such as the ones you worked on in your common PR https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1870